src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import cn from 'classnames';
+import { Filter } from '../../enums/Filter';
+
+type Props = {
+  filter: Filter,
+  showClearButton: boolean,
+  itemsCount: number;
+  setFilter: React.Dispatch<React.SetStateAction<Filter>>,
+  deleteAllTodos: () => Promise<void>
+};
+const TodoFooter: React.FC<Props> = React.memo(({
+  setFilter,
+  filter,
+  showClearButton,
+  itemsCount,
+  deleteAllTodos,
+}) => {
+  const onClickHandler = (value: Filter) => () => {
+    setFilter(value);
+  };
+
+  const countText = `${itemsCount} item${itemsCount > 1 ? 's' : ''} left`;
+
+  const clearCompletedHandler = () => deleteAllTodos();
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {countText}
+      </span>
+
+      <nav className="filter">
+        <a
+          href="src/components/TodoHeader#/"
```

Comment:
wrong href, is supoosed to 
```suggestion
          href="#/"
```